consumer/consumerdata/consumerdata.go

```diff
@@ -16,10 +16,10 @@
 package consumerdata
 
 import (
-	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
-	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
-	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
-	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+	commonpb "github.com/census-instrumentation/opencensus-proto/gen-gogo/agent/common/v1"
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-gogo/metrics/v1"
```

Comment:
github.com/census-instrumentation/opencensus-proto/gen-gogo doesn't seem to exist. So how does this compile?

Reply:
I guess this is the magic: https://github.com/open-telemetry/opentelemetry-service/pull/168/files#diff-37aff102a57d3d7b797f152915a6dc16R55

Reply:
Ah, right, I missed that.